cart.html

```diff
@@ -1,59 +1,62 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
-
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Cart</title>
-</head>
-
-<body>
-  <div id="main-container">
-    <div id="item-container">
-      <h1>Shopping cart</h1>
-
-      <div class="shopping-cart-line-start"></div>
-
-
-      <div id="cart-item-container">
-        <section id="product-item-cart">Product</section>
-        <section id="price-item-cart">Price</section>
-        <section id="quantity-item-cart">Quantity</section>
-        <section id="subtotal-item-cart">Subtotal</section>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Cart</title>
+  </head>
+
+  <body>
+    <div id="main-container">
+      <div id="item-container">
+        <div id="title-btn-container">
+          <button id="home-btn">
+            <a href="index.html"
+              ><img
+                src="/assets/home-page/svgs/home-icon-silhouette-svgrepo-com.svg"
+                alt="Home icon"
+            /></a>
+          </button>
+          <h1>Shopping cart</h1>
+        </div>
+
+        <div class="shopping-cart-line-start"></div>
+
+        <div id="cart-item-container">
+          <section id="product-item-cart">Product</section>
```

Comment:
Section has a larger scope usually. It's an entire section of a (web)page.

What other type of elements could fit here?

Reply:
we could use a `<div>`?